views/chunks.sql

```diff
@@ -0,0 +1,43 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE and LICENSE for copyright and licensing information.
+
+DROP VIEW IF EXISTS chunks_ts, chunks_tstz;
+
+-- Show time range, hypertable, and tablespace for chunks belonging to
+-- hypertables that have a time dimension.
+--
+-- This do not show chunks belonging to hypertables that use an
+-- integer time dimension.
+CREATE OR REPLACE VIEW chunks_tstz AS
+SELECT format('%1$I.%2$I', ch.schema_name, ch.table_name)::regclass AS chunk
+     , format('%1$I.%2$I', ht.schema_name, ht.table_name)::regclass AS hypertable
+     , tstzrange(CASE WHEN sl.range_start = -9223372036854775808 THEN NULL
```

Comment:
SHouldn't this be -inf?

Reply:
I mean, shouldn't this just make a range with (-inf, upper)?

Reply:
I picked `NULL` because it has the same effect. Otherwise, I have no strong opinion.

```
mats=# SELECT tsrange('2020-01-01', NULL) @> tsrange('2020-02-01', '2020-03-01') ;
 ?column? 
----------
 t
(1 row)

mats=# SELECT tsrange('2020-01-01', NULL) <@ tsrange('2020-02-01', '2020-03-01') ;
 ?column? 
----------
 f
(1 row)
```

Reply:
Ah. interesting, okay. I'm fine with that I guess. I'm kinda surprised by that, but I guess it makes sense.